packages/@webex/plugin-meetings/src/meeting/muteState.ts

```diff
@@ -396,9 +398,15 @@ export class MuteState {
 
     // todo: I'm seeing "you can now unmute yourself " popup  when this happens - but same thing happens on web.w.c so we can ignore for now
     this.state.server.remoteMute = false;
-    this.state.client.localMute = false;
 
-    this.applyClientStateLocally(meeting, 'localUnmuteRequired');
+    // change user mute state to false, but keep localMute true if system is still muted
+    this.muteLocalStream(meeting, false, 'localUnmuteRequired');
+    if (this.type === AUDIO) {
+      this.state.client.localMute = meeting.mediaProperties.audioStream?.systemMuted;
```

Comment:
would it maybe be better/safer to read `muted` property from webrtc-core instead of `systemMuted`? it would be more future proof in case anything else (not just system mute) affects the `muted` prop or in case the call to setUserMuted that's done by muteLocalStream fails for some reason. It would also be more consistent with what we do in `handleLocalStreamMuteStateChange()`

Reply:
Yeah, I'll make that change (and also update the comment).